client/my-sites/upgrades/checkout-thank-you/personal-plan-details.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import find from 'lodash/find';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import CustomDomainPurchaseDetail from './custom-domain-purchase-detail';
+import i18n from 'lib/mixins/i18n';
```

Comment:
Let's use 'lib/mixins/i18n/localize' here. It'll wrap the original component and passes through all props plus i18n methods like translate.

```javascript
const PersonalPlanDetails = ( { translate, selectedSite, sitePlans } ) => {
   //...
} );
export default localize( PersonalPlanDetails );
```